### PR TITLE
[FROM-SCRATCH]  Add example functions

### PR DESCRIPTION
**Pull request by aculligan**
* Added example functions to resize, translate, and get user data in ticket_sidebar.js.
* Made changes to main.hdbs and en.json for example.
* Updated comments in index.js to follow [airbnb's style guide.](https://github.com/airbnb/javascript#comments)

@danielbreves 

scaffold

### CODE REVIEW

File: src/javascripts/ticket_sidebar.js [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+    // Get user function
+    const getUser = this.getCurrentUser();
+    
+    // Resize function
+    const resizeApp = (appHeight, appWidth) => client.invoke('resize', { height: appHeight, width: appWidth });
+
+    // Translate function
+    const translate = (keyword) => I18n.t(keyword);
+
     this.view = new View({ afterRender: () => {
```
Comment (danielbreves) [inline src/javascripts/ticket_sidebar.js]: `afterRender` will be called whenever [`this.view.switchTo`](https://github.com/aculligan/app_scaffold/blob/3463024363bb415383eeb2364499ddc270a5c6f6/lib/javascripts/view.js#L16) is called. I don't think we'd want to translate and append the message on `afterRender`.

File: src/javascripts/ticket_sidebar.js [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
 class TicketSidebar {
   constructor(client, data) {
     this.client = client;
     this._metadata = data.metadata;
     this._context = data.context;
-
     this.storage = new Storage(this._metadata.installationId);
+
+    // Get user function
+    const getUser = this.getCurrentUser();
```
Comment (danielbreves) [inline src/javascripts/ticket_sidebar.js]: This isn't being used.

File: src/javascripts/ticket_sidebar.js [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+    
+    // Resize function
+    const resizeApp = (appHeight, appWidth) => client.invoke('resize', { height: appHeight, width: appWidth });
+
+    // Translate function
+    const translate = (keyword) => I18n.t(keyword);
```
Comment (danielbreves) [inline src/javascripts/ticket_sidebar.js]: Not sure I see the benefit of this alias.

File: src/javascripts/ticket_sidebar.js (hunk 2 above)
Comment (aculligan) [inline src/javascripts/ticket_sidebar.js]: Not in this example app, but thought having an alias would be nice.

File: src/javascripts/ticket_sidebar.js (hunk 3 above)
Comment (aculligan) [inline src/javascripts/ticket_sidebar.js]: Just quicker to use `translate()` over `I18n.t()`

File: src/javascripts/ticket_sidebar.js (hunk 1 above)
Comment (aculligan) [inline src/javascripts/ticket_sidebar.js]: We could translated prior to `afterRender` is called then append the message after?

**Pull request closed without merging**